src/main/scala/com/cloudera/sparkts/ARIMA.scala

```diff
@@ -607,7 +602,6 @@ class ARIMAModel(
       // copy into results
       results(-(d + nFuture) to -1) := forwardIntegrated
     }
-    results
```

Comment:
Though actually, we're still returning results either way.

I think it would be better to just remove what's currently the `if` block and surround the else with `if (d != 0)` and then return results at the end.